.gitignore

```diff
@@ -1,7 +1,10 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
-*/node_modules
+*/node_modules/*
```

Comment:
Shouldn't it ignore  node_modules in any directory by default?

Reply:
it was buggin out that seemed to fix it tho